examples/annotation_import/image.ipynb

```diff
@@ -20,12 +20,12 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    mal_label = copy.deepcopy(label)
</pre><p>You don't have to make a copy since you are done with the original label after importing it. You can just overwrite the existing feature schema ids.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>